news/1120.feature

```diff
@@ -0,0 +1 @@
+Format zcml files with collective.zpretty. Add zpretty Github workflow. @tiberiuichim
```

Comment:
@tiberiuichim if I recall correctly there is an "internal" category that we can use for this.